src/components/manage/Tiles/Text/Edit.jsx

```diff
@@ -302,7 +307,12 @@ export default class Edit extends Component {
               <React.Fragment>
                 <div className="separator" />
                 <Button.Group>
-                  <Button icon basic onClick={this.openCustomTileMenu}>
+                  <Button
+                    icon
+                    basic
+                    className="show-hidden-tiles"
```

Comment:
@rodfersou is "show-hidden-tiles" an existing class in our css?

Reply:
I just use this class to fire the button in the test